test/cvodes_dns.jl

```diff
@@ -118,29 +114,23 @@ function cvodes(f, fS, t0, y0, yS0, p, reltol, abstol, pbar, t::AbstractVector)
     tret = [t0]
     yret = similar(y0)
     ysret = similar(yS0)
-    yS0n = [Sundials.NVector(yS0[:, j]) for j in 1:Ns]
+    yS0n = [Sundials.NVector(yS0[:, j]) for j = 1:Ns]
     yS0nv = [N_Vector(n) for n in yS0n]
     pyS0 = pointer(yS0nv)
-    crhs = Sundials.@cfunction(
-        cvrhsfn,
-        Cint,
-        (Sundials.realtype, N_Vector, N_Vector, Ref{CVSData})
-    )
-    csensrhs = Sundials.@cfunction(
-        cvsensrhsfn,
-        Cint,
-        (
-            Cint,
-            Sundials.realtype,
-            N_Vector,
-            N_Vector,
-            N_Vector_S,
-            N_Vector_S,
-            Ref{CVSData},
-            N_Vector,
-            N_Vector,
-        )
-    )
+    crhs = Sundials.@cfunction(cvrhsfn,
+                               Cint,
+                               (Sundials.realtype, N_Vector, N_Vector, Ref{CVSData}))
+    csensrhs = Sundials.@cfunction(cvsensrhsfn,
+                                   Cint,
+                                   (Cint,
+                                    Sundials.realtype,
+                                    N_Vector,
+                                    N_Vector,
+                                    N_Vector_S,
+                                    N_Vector_S,
+                                    Ref{CVSData},
+                                    N_Vector,
+                                    N_Vector))
```

Comment:
why is this all one per line?

Reply:
it's one per line in the original source

Reply:
yeah, that was a mistake of an original formatter run. Can we make it match https://github.com/SciML/SciMLStyle#functions ?

Reply:
you would have to do that yourself initially since `join_lines_based_on_source=true` won't join lines for you

Reply:
okay, thanks. So it won't force this code, but it won't reverse it either?